src/main/java/com/amazon/ion/impl/IonCursorBinary.java

```diff
@@ -1887,13 +1903,80 @@ private boolean slowIsDelimitedEnd_1_1() {
         return false;
     }
 
+    /**
+     * Loads the presence bitmap for the given signature. Upon calling this method, `peekIndex` must point to the
+     * first byte in the argument encoding bitmap (if applicable for the signature).
+     * @param signature the signature of the macro invocation for which to read a presence bitmap.
+     * @return the presence bitmap.
+     */
+    protected PresenceBitmap loadPresenceBitmap(List<Macro.Parameter> signature) {
```

Comment:
Moved from IonReaderContinuableCoreBinary, and corrected to perform `readFrom` and `validate` even if the AEB byte size is 0. This ensures that the in-memory PresenceBitmap is set properly for ExactlyOne arguments.